src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+import { NavLink as RouterNavLink, NavLinkProps as RouterNavLinkProps } from 'react-router-dom';
+import {
+  Flex,
+  NavLink as ThemeUiNavLink,
+  NavLinkProps as ThemeUiNavLinkProps,
+  Box,
+} from 'theme-ui';
+import { Message } from 'retranslate';
+
+import {
+  PERMISSIONS_REQUIRED_FOR_ADD_TEST_TO_IDENTIFIER_PAGE,
+  PERMISSIONS_REQUIRED_FOR_USER_CREATION_PAGE,
+  ADD_TEST_TO_IDENTIFIER_PATH,
+  USER_CREATION_PATH,
+  HOME_PATH,
+} from '../paths';
+import { useAuthentication } from '../authentication';
+
+const CombinedNavLink = ThemeUiNavLink as FC<ThemeUiNavLinkProps & RouterNavLinkProps>;
+
+const NavLink: FC<RouterNavLinkProps> = ({ to, children }) => {
+  return (
+    <CombinedNavLink p={2} as={RouterNavLink} to={to}>
+      {children}
+    </CombinedNavLink>
+  );
+};
+
+export const Navigation = () => {
+  const { hasPermission: userHasPermission, signOut } = useAuthentication();
```

Comment:
In the future, let's change the `hasPermission` function instead

Reply:
Good point, let's do that.